deps/rabbit/src/rabbit_networking.erl

```diff
@@ -61,6 +61,14 @@
 -define(FIRST_TEST_BIND_PORT, 49152).
 
 -define(ETS_TABLE, rabbit_listener_ets).
+%% Number of re-try in case of no_epmd_port
+%% it can happen when the DNS is not ready
+%% for example, in Kubernetes during the start-up phase  
+-define(PORT_PLEASE_ATTEMPTS, 10).
+
+%% Wait for retry when erl_epmd:port_please fails
+%% See erl_epmd_port_please
+-define(PORT_PLEASE_ATTEMPTS_WAIT, 20000).
```

Comment:
Should we decrease this value from 20s to e.g. 10s?
In my tests, I never saw number of retries < 8 (`Getting epmd port node 'rabbit', 8 retries left`).
By decreasing this value, the node will start faster if it needs to retry.

Reply:
>Should we decrease this value from 20s to e.g. 10s?

Yes, I think we can go to 5 seconds